tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -366,6 +368,8 @@ func (ls *leaseStresser) attachKeysWithLease(leaseID int64) error {
 		txn := &pb.TxnRequest{Success: txnPuts}
 		_, err := ls.kvc.Txn(ls.ctx, txn)
 		if err == nil {
+			// since all created keys will be deleted too, the number of operations on keys will be roughly 2x the number of created keys
```

Comment:
from my test, the current rate limiting strategy is pushing the system hard enough. i think we are fine for now.